nifty-core/src/main/java/com/facebook/nifty/ssl/SSLClientConfiguration.java

```diff
@@ -0,0 +1,66 @@
+package com.facebook.nifty.ssl;
+
+import org.jboss.netty.handler.ssl.SslContext;
+import org.jboss.netty.handler.ssl.SslHandler;
+
+import java.io.File;
+
+public class SSLClientConfiguration {
+
+    public static class Builder {
+        Iterable<String> ciphers;
+        String caFile;
+        long sessionCacheSize = 10000;
+        long sessionTimeout = 86400;
+
+        public Builder ciphers(Iterable<String> ciphers) {
+            this.ciphers = ciphers;
+            return this;
+        }
+
+        public Builder caFile(String caFile) {
+            this.caFile = caFile;
+            return this;
+        }
+
+        public Builder sessionCacheSize(long sessionCacheSize) {
+            this.sessionCacheSize = sessionCacheSize;
+            return this;
+        }
+
+        public Builder sessionTimeout(long sessionTimeout) {
+            this.sessionTimeout = sessionTimeout;
+            return this;
+        }
+
+        public SSLClientConfiguration build() {
+            return new SSLClientConfiguration(ciphers, caFile, sessionCacheSize, sessionTimeout);
+        }
+    }
+
+    public final Iterable<String> ciphers;
+    public final String caFile;
+    public final long sessionCacheSize;
+    public final long sessionTimeout;
+
+    public SSLClientConfiguration(Iterable<String> ciphers, String caFile, long sessionCacheSize, long sessionTimeout) {
+        this.ciphers = ciphers;
+        this.caFile = caFile;
+        this.sessionCacheSize = sessionCacheSize;
+        this.sessionTimeout = sessionTimeout;
+    }
+
+    public SslHandler initializeHandler() throws Exception {
```

Comment:
`initialize` should as if it receives and object and modifies it in place to set the required state. Here I think `get` or `create` would fit better